packages/netdog/generate-network-config.service

```
@@ -1,9 +1,14 @@
 [Unit]
 Description=Generate network configuration
+DefaultDependencies=no
 # Block manual interactions with this service, since it could leave the system in an
 # unexpected state
 RefuseManualStart=true
 RefuseManualStop=true
+Before=network-pre.target
+After=\x2ebottlerocket-netdog.mount
+Requires=\x2ebottlerocket-netdog.mount
```

These are redundant with `RequiresMountsFor`:
```suggestion
After=\x2ebottlerocket-netdog.mount
Requires=\x2ebottlerocket-netdog.mount
```